packages/ui/src/components/Faucet.tsx

```diff
@@ -36,7 +36,7 @@ class Faucet extends RefreshableComponent<Props, {}> {
           requests={faucet.faucetRequests}
           onRefresh={() => faucet.refreshFaucetRequestStatus()}
         />
-        <CliHint requests={faucet.faucetRequests} />
+        {/* <CliHint requests={faucet.faucetRequests} /> */}
```

Comment:
Please remove it and CliHint class as well.

Reply:
Ok

Reply:
Ok